src/main/resources/application.yml

```diff
@@ -10,7 +10,7 @@ spring:
       #监控统计拦截的filters
       filter: stat
       #mysql驱动
-      driver-class-name: com.mysql.jdbc.Driver
+      driver-class-name: com.mysql.cj.jdbc.Driver
```

Comment:
👎 *but the version of mysql driver is : `5.7.24` so that this code isn't support it (`in fact it's only support the version : 6.0+`) .  for more informations please refer to my blog : [go](https://github.com/YUbuntu0109/YUbuntu0109.github.io/blob/HexoBackup/source/_posts/JDBC-%E5%B8%B8%E8%A7%81%E9%94%99%E8%AF%AF%E5%88%86%E6%9E%90.md)*